src/components/Modal/__stories__/modal.stories.mdx

```diff
@@ -0,0 +1,308 @@
+              <TextField title="Your Name" size={TextField.sizes.MEDIUM} placeholder="John Doe" />
+              <TextField title="Your Name" size={TextField.sizes.MEDIUM} placeholder="John Doe" />
+              <TextField title="Your Name" size={TextField.sizes.MEDIUM} placeholder="John Doe" />
+              <TextField title="Your Name" size={TextField.sizes.MEDIUM} placeholder="John Doe" />
+              <TextField title="Your Name" size={TextField.sizes.MEDIUM} placeholder="John Doe" />
+              <TextField title="Your Name" size={TextField.sizes.MEDIUM} placeholder="John Doe" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+              <TextField title="Email" size={TextField.sizes.MEDIUM} placeholder="email@monday.com" />
+            </form>
+            <ModalFooter>{modalButtons}</ModalFooter>
+          </Modal>
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+### Modal with custom Header
+
+Header with description and icon
+
+<Canvas>
+  <Story name="Modal with custom Header">
+    {() => {
+      const { openModalButton, modalButtons, show, closeModal, ref } = useModalHelper({
+        openButtonTitle: "Default"
```

Comment:
It's better to change name of button "Default" - to something more related to story: "Custom Header"